MAVProxy/modules/mavproxy_link.py

```diff
@@ -137,11 +143,16 @@ def link_add(self, device):
             return False
         if self.settings.rtscts:
             conn.set_rtscts(True)
-        conn.linknum = len(self.mpstate.mav_master)
         conn.mav.set_callback(self.master_callback, conn)
         if hasattr(conn.mav, 'set_send_callback'):
             conn.mav.set_send_callback(self.master_send_callback, conn)
         conn.linknum = len(self.mpstate.mav_master)
+        try:
+            handle = link_components[1]
+            handle = handle.rstrip(')')
+            conn.handle = handle
```

Comment:
Perhaps "label" would be better than "handle"; less confusing with a "file handle" that way...